tests/test_result_visitor.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+
+from robotframework_reportportal.result_visitor import RobotResultsVisitor
+
+
+class ResultVisitorTest(unittest.TestCase):
+    def test_parse_message_empty(self):
+        o = RobotResultsVisitor()
+        self.assertRaises(AttributeError, o.parse_message, 'usual test comment without image')
+        self.assertRaises(AttributeError, o.parse_message, '<img src=\'bad.html.img>')
```

Comment:
use delayed_asserts library if there is no way to avoid multiple asserts within one test.